sympy/printing/printer.py

```diff
@@ -230,7 +230,8 @@ def order(self):
 
     def doprint(self, expr):
         """Returns printer's representation for expr (as a string)"""
-        return self._str(self._print(expr))
+        res=self._str(self._print(expr))
+        return res
```

Comment:
Sorry, just a leftover of "print debugging..." I removed the print but forgot to inline it again, I will do it now along with the removal of the white space that I forgot as well..